src/plugins/lvm-dbus.c

```diff
@@ -2290,13 +2306,14 @@ BDLVMLVdata** bd_lvm_lvs (const gchar *vg_name, GError **error) {
     lv = matched_lvs;
     while (lv) {
         props = get_object_properties (lv->data, LV_CMN_INTF, error);
+        g_free (lv->data);
```

Comment:
Since you're doing proper `g_slist_free_full()` in error cases and the final success case now, this would result in double-free in the error cases since you're freeing the original list, not the iterated one (which is correct). 

Suggesting to remove this line.

Reply:
Good catch.

Reply:
Removed in updated commit.